nservicebus/messaging/data-distribution.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Data distribution
+summary: Why asynchronous messaging is not a good fit for data distribution scenarios
+reviewed: 2020-05-21
+redirects:
+- nservicebus/azure/data-distribution
+- samples/routing/data-distribution
+---
+
+Data distribution refers to a pattern where asynchronous messages (i.e. using NServiceBus) are used to deliver data to multiple targets in some coordinated fashion.
+
+A common example of data distribution scenario is having cached data on multiple scaled-out web servers, and then attempting to use messages delivered to all web servers in a cache-busting fashion, so that each web server drops its current cache entries, and then is forced to retrieve fresh data from the database.
+
+## Timing and reliability
+
+The goal of data distribution is generally to cause a change in data and/or behavior in multiple locations at the same time. Messages sent with NServiceBus, however do not care if they are received or processed at the same time. There is no urgency to act _now_ with a message sent with NServiceBus. Because asynchronous messages are built for eventual consistency, each receiver would be free to process it at their leisure, whenever it gets around to handling that mesage in the queue.
```

Comment:
```suggestion
The goal of data distribution is generally to cause a change in data and/or behavior in multiple locations at the same time. Messages sent with NServiceBus, however do not care if they are received or processed at the same time. There is no urgency to act _now_ with a message sent with NServiceBus. Because asynchronous messages are intended for eventual consistency, each receiver would be free to process it at their leisure, whenever it gets around to handling that message in the queue.
```